Tools/cases_generator/tier1_generator.py

```diff
@@ -158,7 +158,10 @@ def generate_tier1(
         out.emit(f"next_instr += {inst.size};\n")
         out.emit(f"INSTRUCTION_STATS({name});\n")
         if inst.is_target:
-            out.emit(f"PREDICTED({name});\n")
+            out.emit(f"PRED_{name}:\n")
```

Comment:
```suggestion
            out.emit(f"PRED_{name}:;\n")
```
No need for the braces below, it you add a semicolon here